### DISCUSSION TITLE
assets behavior mac vs ios

### DISCUSSION THREAD

**Discussion by jean-david-ruvini** [flet-dev/flet#1692] [General]
Hi,
I am having trouble displaying images in my app.

I have them in the "assets" folder next to my main.py.

I specify it this way:

ft.app(target=main, assets_dir="assets")

And load them this way:

ft.Image(
    src='./assets/'+se.merchants[0].logo,
    width=frontend_config.MERCHANT_IMG_WIDTH,
    height=frontend_config.MERCHANT_IMG_HEIGHT,
    fit=ft.ImageFit.CONTAIN
)

This works fine on IOS when running my app from "flet run --ios"
But the image doesn't show when I run the app on my macbook with "flet run"

If I change to this:
src=se.merchants[0].logo,

I have the opposite behavior: the image doesn't show with "flet run --ios" but shows with "flet run"

what am I missing?
Thanks for any help!
JD

**Comment by FeodorFitsner** [comment]
Looking into that...

**Comment by FeodorFitsner** [comment]
It's a bug! Good catch, thanks!
Filed https://github.com/flet-dev/flet/discussions/1693
